.idea/httpRequests/http-requests-log.http

```diff
@@ -0,0 +1,197 @@
+POST https://anfisa.forome.dev/lihander-dev/redeploy
```

Comment:
I didn't check functionality but switching between GENE and VARIANT views doesn't work.